packages/frontend/src/components/AllBids/SettledBidsList.tsx

```diff
@@ -31,6 +33,10 @@ export const SettledBidsList = ({ search }: SettledBidsListProps) => {
   )
 
   const filteredBids = useMemo(() => filterBids(settledBids, searchFunc), [settledBids, searchFunc])
+  const goldenTicketWinner = useMemo(
+    () => raffleWinners?.[0] && bids.find((bid) => bid.bidderID.eq(raffleWinners[0])),
+    [bids, raffleWinners]
+  )
```

Comment:
We can move that logic to `filterBids` function to return separate `goldenTicket` field in `Bids` struct.